<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FenPrincipale</class>
 <widget class="QMainWindow" name="FenPrincipale">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1319</width>
    <height>717</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FenPrincipale</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="f_barreReglages">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>201</width>
      <height>631</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>201</width>
       <height>631</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="vl_barreReglages_2">
      <item>
       <widget class="QFrame" name="f_chargerImg">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QPushButton" name="pb_chargerImg">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>181</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Charger image</string>
         </property>
        </widget>
        <widget class="QComboBox" name="cb_choixImg">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>181</width>
           <height>27</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="la_choixImg">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>181</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Choix image:</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pb_agrandir">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>100</y>
           <width>181</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Agrandir image (droite)</string>
         </property>
        </widget>
        <widget class="QLabel" name="la_masq">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>150</y>
           <width>131</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Choix du masque :</string>
         </property>
        </widget>
        <widget class="QComboBox" name="cb_choixMasq">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>170</y>
           <width>111</width>
           <height>27</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pb_masque">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>170</y>
           <width>61</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>Recalc</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="f_seuillage">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QComboBox" name="cb_seuill">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>111</width>
           <height>27</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="la_seuill">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>131</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Choix du seuillage :</string>
         </property>
        </widget>
        <widget class="QSlider" name="hs_seuill">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>181</width>
           <height>16</height>
          </rect>
         </property>
         <property name="maximum">
          <number>200</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QPushButton" name="pb_seuill">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>30</y>
           <width>61</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>Recalc</string>
         </property>
        </widget>
        <widget class="QLabel" name="la_valSeuil">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>131</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Valeur seuillage :</string>
         </property>
        </widget>
        <widget class="QLabel" name="la_choixAffinage">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>110</y>
           <width>151</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Choix de l'affinage :</string>
         </property>
        </widget>
        <widget class="QComboBox" name="cb_choixAffinage">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>130</y>
           <width>181</width>
           <height>27</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pb_Affiner">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>160</y>
           <width>81</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Affinage</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="f_affinage">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="la_gradMinFermeture">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>181</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Grad. min. pour fermeture :</string>
         </property>
        </widget>
        <widget class="QSlider" name="hs_gradMinFermeture">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>181</width>
           <height>16</height>
          </rect>
         </property>
         <property name="maximum">
          <number>600</number>
         </property>
         <property name="value">
          <number>200</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QPushButton" name="pb_modeSegments">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>120</y>
           <width>81</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Recalc</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="sp_tailleSegements">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>130</y>
           <width>81</width>
           <height>27</height>
          </rect>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>3</number>
         </property>
        </widget>
        <widget class="QCheckBox" name="cb_colDiff">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>170</y>
           <width>181</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>Colorer différences</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pb_fermeture">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>50</y>
           <width>81</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Fermeture</string>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>100</y>
           <width>171</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Modélisation segments :</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="f_visuels">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>1081</width>
      <height>581</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1081</width>
       <height>581</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="hl_visuels">
      <item>
       <widget class="QTabWidget" name="tw_visuels1">
        <property name="currentIndex">
         <number>6</number>
        </property>
        <widget class="QWidget" name="t_Source1">
         <attribute name="title">
          <string>Tab 1</string>
         </attribute>
         <widget class="QLabel" name="la_source1">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>521</width>
            <height>491</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="t_Seuillage1">
         <attribute name="title">
          <string>Tab 2</string>
         </attribute>
         <widget class="QLabel" name="la_seuillage1">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>521</width>
            <height>491</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="t_carteGradient1">
         <attribute name="title">
          <string>Page</string>
         </attribute>
         <widget class="QLabel" name="la_carteGradient1">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>521</width>
            <height>491</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="t_pentes1">
         <attribute name="title">
          <string>Page</string>
         </attribute>
         <widget class="QLabel" name="la_pentes1">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>521</width>
            <height>491</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="t_affinage1">
         <attribute name="title">
          <string>Page</string>
         </attribute>
         <widget class="QLabel" name="la_affinage1">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>521</width>
            <height>491</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="t_fermeture1">
         <attribute name="title">
          <string>Page</string>
         </attribute>
         <widget class="QLabel" name="la_fermeture1">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>521</width>
            <height>491</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="t_moedelisation1">
         <attribute name="title">
          <string>Page</string>
         </attribute>
         <widget class="QLabel" name="la_modelisation1">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>521</width>
            <height>491</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tw_visuels2">
        <property name="currentIndex">
         <number>6</number>
        </property>
        <widget class="QWidget" name="t_Source2">
         <attribute name="title">
          <string>Tab 1</string>
         </attribute>
         <widget class="QLabel" name="la_source2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>521</width>
            <height>491</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="t_Seuillage2">
         <attribute name="title">
          <string>Tab 2</string>
         </attribute>
         <widget class="QLabel" name="la_seuillage2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>521</width>
            <height>491</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="t_carteGradient2">
         <attribute name="title">
          <string>Page</string>
         </attribute>
         <widget class="QLabel" name="la_carteGradient2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>521</width>
            <height>491</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="t_pentes2">
         <attribute name="title">
          <string>Page</string>
         </attribute>
         <widget class="QLabel" name="la_pentes2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>521</width>
            <height>491</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="t_affinage2">
         <attribute name="title">
          <string>Page</string>
         </attribute>
         <widget class="QLabel" name="la_affinage2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>521</width>
            <height>491</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="t_fermeture2">
         <attribute name="title">
          <string>Page</string>
         </attribute>
         <widget class="QLabel" name="la_fermeture2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>521</width>
            <height>491</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="t_modelisation2">
         <attribute name="title">
          <string>Page</string>
         </attribute>
         <widget class="QLabel" name="la_modelisation2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>521</width>
            <height>491</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="la_avertissement">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>1081</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Si l'image est plus grosse que la taille de la zone alouée ci-dessous,  l'image sera scalée et le résultat faussé, en ce cas cliquez sur agrandir image à gauche.</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="pg_progressionAlgo">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>620</y>
      <width>1081</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1319</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
